test/instances.lean

```diff
@@ -0,0 +1,23 @@
+/-
+A short example to test the behavior of `get_instances`.
+To actually test this, paste this code above the `HACK` lines in `export_json.lean`, as after those
+lines notation becomes badly broken.
+-/
```

Comment:
This test isn't actually executed by CI due to the problem that `export_json` poisons the notation, but a test that can be manually executed is better than no test at all.

Reply:
We could move the hacks to entrypoint.lean